spack.yaml

```diff
@@ -53,7 +58,7 @@ spack:
               'SPACK_{name}_ROOT': '{prefix}'
         projections:
           all: '{name}/{version}'
-          access-om2: '{name}/2024.03.0'
+          access-om2: '{name}/2024.04.0'
```

Comment:
We probably want to update to `2024.06.0` if that is when we're merging the PR.

Reply:
Updated in https://github.com/ACCESS-NRI/ACCESS-OM2/pull/60/commits/6ab1bd7bcd6f0635b462794a92950cca737be1a6 @aidanheerdegen